static-assets/components/cstudio-forms/forms-engine.js

```
@@ -2435,8 +2435,9 @@ const initializeCStudioForms = () => {
 				 * repeat manipulation events
 				 */
 				_renderRepeatBody: function (repeatContainerEl) {
-					var maxOccurs = repeatContainerEl.maxOccurs;
-					var minOccurs = repeatContainerEl.minOccurs;
+					// If value for min/max is not set, use default values (0 for min and '*' for max)
+					var maxOccurs = Boolean(repeatContainerEl.maxOccurs) ? repeatContainerEl.maxOccurs : '*';
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)